src/bacon.rs

```diff
@@ -339,7 +338,7 @@ impl Bacon {
 
         if let Some(workspace_folders) = workspace_folders {
             for folder in workspace_folders.iter() {
-                let mut folder_path = PathBuf::from(folder.uri.path());
+                let mut folder_path = folder.uri.to_file_path().unwrap();
```

Comment:
I think I'd prefer this error to be bubbled up with a `?`, but it will require some refactoring of `diagnostics()`, `diagnostics_vec()` and all their callers. 

For now I am happy to just use expect and panic with a nicer error:

```suggestion
                let mut folder_path = folder.uri.to_file_path().expect("the workspace folder sent by the editor is not a file path");
```